flink-runtime/src/main/java/org/apache/flink/runtime/metrics/dump/MetricQueryService.java

```diff
@@ -165,11 +199,17 @@ static String replaceInvalidChars(String str) {
 	 * @param resourceID resource ID to disambiguate the actor name
 	 * @return actor reference to the MetricQueryService
 	 */
-	public static ActorRef startMetricQueryService(ActorSystem actorSystem, ResourceID resourceID) {
+	public static ActorRef startMetricQueryService(
+		ActorSystem actorSystem,
+		ResourceID resourceID,
+		MetricRegistryConfiguration configuration) {
```

Comment:
yet again, just pass in the maximumFramesize. It doesn't make sense that a user has to provide a  MetricRegistryConfiguration to start the MQS actor.